src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -82,9 +81,11 @@ class Comment extends React.Component<Props, State> {
 
     approvalCommentFormSubmitHandler = (): void => this.setState({ isInputOpen: false, isEditing: false });
 
-    handleUpdate = (args: any): void => {
-        const { onEdit = noop } = this.props;
-        onEdit(args);
+    handleUpdate = ({ id, text, hasMention }: { hasMention: boolean, id: string, text: string }): void => {
+        const { onEdit, permissions } = this.props;
+        if (onEdit) {
```

Comment:
It looks like this may always be defined. Can we continue default it to `noop` via `defaultProps`, if not?

Reply:
This followed what the onDelete handler was doing, but I will change them both

Reply:
🤔 now we can't hide the menu based on that function being undefined. I guess I can check for `=== noop` instead.

Reply:
I didn't realize there was a case for it being undefined. Feel free to leave it as-is, if needed.